__tests__/kata3.numberToReversedDigits.test.js

```diff
@@ -5,5 +5,6 @@ describe('numberToReversedDigits', () => {
     expect(numberToReversedDigits(12345)).toEqual([5, 4, 3, 2, 1]);
     expect(numberToReversedDigits(6789)).toEqual([9, 8, 7, 6]);
   })
+  // review
```

Comment:
It isn't recommended to have line breaks after a block of code opens, or before a block of code closes, and in any scenario you should never have more than one line break.